cmd/root.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kf5i/k3ai-core/pkg/k8s/kctl"
+	"github.com/kf5i/k3ai-core/pkg/plugins"
+	"github.com/spf13/cobra"
+)
+
+const k3aiBinaryName = "k3ai-client"
```

Comment:
`k3ai-cli` would be better !

Reply:
yes agreed.

Reply:
If you could merge #4 , I'll make the required changes as there are a few places where this needs to be addressed

Reply:
>If you could merge #4 

Done!